client/src/test/java/org/eclipse/hono/client/impl/VertxSupport.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.client.impl;
+
+import org.mockito.ArgumentMatchers;
+
+import io.vertx.core.Handler;
+
+/**
+ * Argument matchers for the use with vert.x.
+ */
+public final class VertxSupport {
```

Comment:
If this class is supposed to provide ArgumentMatchers, then why not call it `ArgumentMatchers`?

Reply:
That is what I named it at first. Looking at a few other locations I found a few more targets to work on (mock(), ArgumentCaptor), … I thought it might make more sense to put all of those into `VertxSupport`, rather than creating three different classes.

Reply:
Then what about `VertxMockSupport` ?

Reply:
Done. Also added two more variants of this.